### DISCUSSION TITLE
blank page as default template

### DISCUSSION THREAD

**Discussion by Wladefant** [personalizedrefrigerator/js-draw#36] [Ideas & Feature Requests]
I would suggest using a blank page as the default template for js draw like all the other programs like onenote:
![image](https://github.com/personalizedrefrigerator/js-draw/assets/79611093/ebbf5bf8-6a15-4aa6-aac8-d948db1ee9ef)
samsung notes:
![SmartSelect_20230919_233344_Samsung Notes](https://github.com/personalizedrefrigerator/js-draw/assets/79611093/3c5fe36a-e3ad-40d5-923a-e51c95be4742)

especially if you are calling it "drawing"
 
but either way i can see that that is easy to change. Still... If you need a grid you will turn it on, otherwise a drawing does not need lines...

**Comment by personalizedrefrigerator** [comment]
My reasoning behind using a grid pattern by default:
- If a user turns on "touchscreen panning", exits `js-draw`, then creates a new `js-draw` image, "touchscreen panning" will still be enabled. If there is no background pattern, attempting to draw with touch gives no feedback (except perhaps the sometimes difficult-to-see scrollbars). If there is a background pattern, drawing with touch visibly moves the background.

A drawing title input (like OneNote's) or other default content would also work to resolve the above feedback issue.

Note that changing background settings in the "page" menu also change the template used for new images. As such, unchecking "grid" then saving means that the next image created will not be created with a grid background.

**Reply by personalizedrefrigerator** [reply]
**Note:** The default template was ultimately changed to a blank page in the main Joplin app.
